docs/manual/manual_circuit.md

```diff
@@ -622,7 +622,7 @@ print("Total gate count =", circ.n_gates)
 print("Circuit depth =", circ.depth())
 ```
 
-As characteristics of a {py:class}`~pytket.circuit.Circuit` go, these are pretty basic. In terms of approximating the noise level, they fail heavily from weighting all gates evenly when, in fact, some will be much harder to implement than others. For example, in the NISQ era, we find that most technologies provide good single-qubit gate times and fidelities, with two-qubit gates being much slower and noisier [^cite_arut2019]. On the other hand, looking forward to the fault-tolerant regime we will expect Clifford gates to be very cheap but the magic $T$ gates to require expensive distillation procedures [^cite_brav2005] [^cite_brav2012].
```

Comment:
I was getting a warning that this citation doesn't appear in the refernces. Just removed it for now as I'm not sure which paper it was.